hack/quickstart/quickstart-openstack.md

```diff
@@ -0,0 +1,162 @@
+## OpenStack Quickstart
+
+These instructions have been tested on vexxhost.com. They should work for other providers though.
+
+Ensure you have a working nova/glance/openstack tool that can authenticate before running these steps. Test this with `openstack image list` and `openstack server list`.
+
+Disclaimer: This is "best effort maintained". This means that the instructions below are not actively maintained and may be out of date.
```

Comment:
fixed the bold nit. I added a "Owners" section at the end of this doc, I assume you didn't mean changing https://github.com/kubernetes-incubator/bootkube/blob/master/OWNERS, right?